### PR TITLE
CI: enable sqlite system tests

### PR DESCRIPTION
**Pull request by edsantiago**
In setup, write a containers.conf.d file with db_backend
as specified in .cirrus.yml.

This is actually much scarier and more achy-breaky than
merely "sqlite system tests": it enables sqlite in e2e
tests. ("But wait, we already do that!" -- no, not really.
sqlite in e2e is being done via --db-backend option, and
some podman commands in e2e do not use the standard options.
See #17904.

This is unlikely to get merged any time soon (March, maybe
even April) because sqlite is still too fragile; this will
trigger more flakes than are currently acceptable. Also,
the nasty auto-update flake seems to trigger much more
reliably with sqlite. We need that one fixed.

Signed-off-by: Ed Santiago <santiago@redhat.com>
```release-note
None
```

### CODE REVIEW

**Comment by edsantiago** [comment]
Marked WIP because I'm going to be repeatedly running tests, and waiting for #17937 and #17953, then repeatedly running more tests.

**Comment by edsantiago** [comment]
heads-up, github CI reporting seems to be stalled. Job is done, all green, but as of the moment I write this github is still showing jobs queued.

**Comment by edsantiago** [comment]
@cevich this job finished (green) many hours ago. Is there any way to tell Github to catch up, or do we have to resubmit all jobs?

**Comment by cevich** [comment]
Whoa, that's weird.  I don't remember ever seeing that behavior before.  Looking at the timer, it's going on 6-hours, so I'd bet it will never be marked as completed.  Since clearly they're all green, I would just ask for this to be manually merged.  Unless you think re-running would be faster :disappointed:

**Comment by edsantiago** [comment]
I don't mind repushing - this is a scary PR, and more testing is good. It's just, all PRs from this morning are stuck because of the github hiccup. I was wondering if there was a fix.

**Comment by edsantiago** [comment]
Blocked by #17904

**Comment by edsantiago** [comment]
@cevich @containers/podman-maintainers this is ready to go. I hand-checked all logs, all flakes are the usual ones.

**Review by vrothberg — APPROVED** [review]
/lgtm
/hold

Waiting for a head nod from @cevich

**Pull request merged**
